index.css

```diff
@@ -0,0 +1,526 @@
+.topnav {
+  color: white;
+  background-color: #3e3c3c;
+  position: absolute;
+  width: 380px;
+  height: 92px;
+  left: 0;
+  top: 0;
+  padding-right: 21px;
+}
+
+.topnav #myLinks {
+  display: none;
+}
+
+a {
+  float: left;
+  color: white;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+a .active {
+  position: relative;
+  width: 77px;
+  height: 24px;
+  left: 28px;
+}
+
+.menor {
+  width: 54px;
+  height: 16px;
+  left: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  color: #fff;
+  padding-left: 40px;
+}
+
+.topnav a.icon {
+  float: right;
+}
+
+.topnav a:hover {
+  background-color: #ddd;
+  color: #3c3a39;
+}
+
+/* end of tobnav */
+.roger {
+  width: 351px;
+  height: 380px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  position: relative;
+  top: 80px;
+  left: -8px;
+  background-color: #1c1a19;
+}
+
+.pad {
+  position: relative;
+  color: #ff6b00;
+
+  /* bottom: 30px; */
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. :+1: